# Rescuer
[![Gem Version](https://badge.fury.io/rb/rescuer.png)](http://badge.fury.io/rb/rescuer)
[![Build Status](https://travis-ci.org/ms-ati/rescuer.png)](https://travis-ci.org/ms-ati/rescuer)
[![Dependency Status](https://gemnasium.com/ms-ati/rescuer.png)](https://gemnasium.com/ms-ati/rescuer)
[![Code Climate](https://codeclimate.com/github/ms-ati/rescuer.png)](https://codeclimate.com/github/ms-ati/rescuer)
[![Coverage Status](https://coveralls.io/repos/ms-ati/rescuer/badge.png)](https://coveralls.io/r/ms-ati/rescuer)

A functional abstraction of rescuing exceptions inspired by Scala's [Try][1] (an [explanation][2]).

[1]: http://www.scala-lang.org/api/2.10.3/index.html#scala.util.Try
[2]: http://danielwestheide.com/blog/2012/12/26/the-neophytes-guide-to-scala-part-6-error-handling-with-try.html

## Usage

```ruby


```